### ISSUE TITLE
Byte type

### ISSUE THREAD

**Issue by rob-stewart** [tvaneerd/cpp17_in_TTs#26]
I've never used a type with "byte" in its name except the one for Win32 APIs: BYTE. However, I have used char, uint8_t, etc., to mean a byte type. Having a byte_t would be better as it would be semantically correct and would be portable to any remaining platforms with non-8-bit bytes.

**Comment by tvaneerd** [comment]
Yep, it is a good idea. There real question is whether you find std::byte works as expected or not, and whether it suits your uses.
